account_invoice_fiscal_position_update/tests/test_inv_fiscal_pos_update.py

```diff
@@ -127,10 +127,9 @@ def test_fiscal_position_id_change(self):
             fp_account.account_dest_id,
             "The account revenue of invoice line must be changed by fiscal position",
         )
-        # Test warning due to lines without product
+        # Test warning due to lines without product and description
         self.invoice_line_model.with_context(check_move_validity=False).create(
             {
-                "name": "Line without product",
```

Comment:
Could you please add a check to verify if `len(lines_without_product) != len(invoice_lines)`?

Reply:
Done! Please, let me now if it's ok.

Reply:
`l.product_id.exists()` -> `not l.product_id.exists()`?

Reply:
Indeed. Now it's fixed.